airbyte-integrations/connectors/source-news-api/source_news_api/spec.yaml

```diff
@@ -0,0 +1,181 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/news-api
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: News Api Spec
+  type: object
+  required:
```

Comment:
The term is not required?

Reply:
It's not required by the API so I don't see any reason to require it here.